### DISCUSSION TITLE
Intrigue v0.1.6: Link support, new icon

### DISCUSSION THREAD

**Discussion by shaunabanana** [shaunabanana/intrigue#12] [Announcements]
This release adds support for automatically recognizing URLs, as well as `zotero://` item links. Now you can use Quick Copy in Zotero to get an item link (as described [here](https://zoteromusings.wordpress.com/2013/04/23/zotero-item-uris-from-client/)), and Intrigue will redirect to the item when the link is clicked.

On a related note, I've just published [Snowball](https://github.com/shaunabanana/snowball), a tool that enables you to quickly find and screen papers using the "snowballing" method. It saves you the pain of searching citation/reference lists by hand. Since the two apps are quite related, I've redesigned the icon of Intrigue along with the new Snowball icon. Now the two sits together better.

Happy researching!

P.S. This time, the multi-platform releases are all built on my M1 Mac. Please let me know if anything goes wrong.

<hr /><em>This discussion was created from the release <a href='https://github.com/shaunabanana/intrigue/releases/tag/v0.1.6'>Link support, new icon</a>.</em>

**Comment by MAAliy** [comment]
Would it be possible to add pictures and export as docx/pdf capability?
